linux_os/guide/system/software/integrity/crypto/openssl_use_strong_entropy/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+# TODO: The plan is not to need this for RHEL>=8.4
+prodtype: rhel8
+
+title: 'OpenSSL uses strong entropy source'
+
+description: |-
+    To set up an <tt>openssl</tt> wrapper that adds a <tt>-rand /dev/random</tt> option to the <tt>openssl</tt> invocation,
+    save the following shell snippet to the <tt>/etc/profile.d/openssl-rand.sh</tt>:
```

Comment:
I'd say `save the following shell snippet exactly "as is"`.
My thinking is: Someone copy pasting the snippet may change indentation, comments or blank lines.